architectures/lib/commands/check-repositories.yml

```diff
@@ -0,0 +1,16 @@
+---
+
+# © Copyright EnterpriseDB UK Limited 2015-2023 - All rights reserved.
+
+- name: Check for old repositories
+  hosts: all
+
+  tasks:
+  - assert:
+      that: "{{ tpa_2q_repositories is not defined }}"
```

Comment:
I think we should not warn if it's explicitly set to `[]`

Reply:
Done.